pepper-apis/src/main/java/org/broadinstitute/ddp/route/UserRegistrationRoute.java

```diff
@@ -566,33 +564,31 @@ private void initializeProfile(Handle handle, User user, UserRegistrationPayload
         }
     }
 
-    private void queueUserRegisteredEvents(Handle handle, StudyDto studyDto, User operator, User participant) {
-        QueuedEventDao queuedEventDao = handle.attach(QueuedEventDao.class);
-        List<EventConfigurationDto> configs = handle.attach(EventDao.class)
-                .getActiveDispatchConfigsByStudyIdAndTrigger(studyDto.getId(), EventTriggerType.USER_REGISTERED);
+    private void triggerUserRegisteredEvents(Handle handle, StudyDto studyDto, User operator, User participant) {
+        var signal = new EventSignal(
+                operator.getId(),
+                participant.getId(),
+                participant.getGuid(),
+                studyDto.getId(),
+                EventTriggerType.USER_REGISTERED);
+        EventService.getInstance().processAllActionsForEventSignal(handle, signal);
```

Comment:
I updated the implementation to use `EventService`. This means we can now run pretty much any kind of event action, where previously it was only notification emails.